probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/RunConfigurations.scala

```diff
@@ -182,6 +149,65 @@ object RunConfigurations extends IntelliJApi {
     new RunnerSettingsWithProcessOutput(settings)
   }
 
+  private def availableRunConfigurations(
+                                          scope: TestScope
+                                        )(implicit ec: ExecutionContext): java.util.List[ConfigurationFromContext] = {
```

Comment:
Maybe return a scala collection, so that on the use side it doesn't need to be converted

Reply:
Good point